File: Intermediate/algorithms/selectionSort.c
void selectionSort(int arr[], int size)
{
    for (int i=0; i<size-1; i++)
    {
        int minI = i;
        for (int j=i+1; j<size; j++)
        {
            if (arr[j] < arr[minI])
            {
                minI = j;
            }
        }
        int swap = arr[minI];
        arr[minI] = arr[i];
        arr[i] = swap;
    }
}

File: Intermediate/algorithms/selection_sort.py
def selection_sort(array):
    for i in range(len(array)):
        tmp=min(array[i:len(array)])
        array.remove(min(array[i:len(array)]))
        array.insert(i,tmp)
    return array

n=int(input('Enter Size of Array: '))
arr=[int(input('Enter Element %d : '%(i+1))) for i in range(n)]

print ('Original Array : ',arr)
print ('Sorted Array : ',selection_sort(arr))
